src/commands/npm/lerna/release.ts

```diff
@@ -144,4 +150,16 @@ export default class Release extends SfdxCommand {
       return { name: pkg.name, version: pkg.getNextVersion() };
     });
   }
+
+  protected async verifySign(pkgInfo: PackageInfo): Promise<void> {
+    const cmd = 'trust:plugins:verify';
+    const argv = `--npm ${pkgInfo.name}@${pkgInfo.nextVersion} ${pkgInfo.registryParam}`;
+
+    this.ux.log(chalk.dim(`sf-release ${cmd} ${argv}`) + os.EOL);
+    try {
+      await this.config.runCommand(cmd, argv.split(' '));
```

Comment:
`runCommand` doesn't return the exit code so we just wrap it in a try catch block.